swupd/cmd/create-pack/main.go

```diff
@@ -74,153 +72,74 @@ func main() {
 		}
 	}
 
-	if *outputDir == "" {
-		tempDir, err := ioutil.TempDir(".", "packs-")
+	var err error
+	var bundles map[string]*swupd.BundleToPack
+
+	if *allBundles {
```

Comment:
It would be really nice to have this moved into the regular Mixer CLI so we're not carrying 2 cli implementations in the same project. Out of scope for this PR, but that should be updated soon IMO.

Reply:
These programs let me test the functionality without having to do all the mixer configuration and steps. I'll probably nuke them once I set up better automated testing for the functionality.

Agree out of scope of this PR to remove them.